packages/image-cloze-association/controller/src/__tests__/index.test.js

```diff
@@ -323,8 +327,8 @@ describe('controller', () => {
           validResponse: {
             score: 1,
             value: [
-              [rhomb, square],
-              [rhomb, square, trapeze],
+              {images: [rhomb, square]},
```

Comment:
Let's add tests where we have {}, or { images: null} etc. (This will require to add sanity checks on the next file).